_build-and-test-all.sh

```diff
@@ -32,4 +33,12 @@ echo 'show dbs' |  ./mongodb-cli.sh -i
 
 ./gradlew :end-to-end-tests:cleanTest :end-to-end-tests:test
 
+if [ "${DATABASE}" == "mysql" ]; then
+  cat "add-database-id-support-to-eventuate-mysql.migration.sql" | ./mysql-cli.sh -i
+fi
+
+${dockerall}Up -P envFile=docker-compose-env-files/db-id-gen.env
```

Comment:
Why run the tests again if the database is not mysql?

Reply:
I will fix